dcm4che-core/src/main/java/org/dcm4che3/data/Attributes.java

```diff
@@ -1671,6 +1848,16 @@ public void setDefaultTimeZone(TimeZone tz) {
         defaultTimeZone = tz;
     }
 
+    public ZoneId getDefaultZoneId() {
+        if (defaultTimeZone != null)
+            return defaultTimeZone.toZoneId();
+
+        if (parent != null)
+            return parent.getDefaultZoneId();
+
+        return null;
```

Comment:
Note the different behavior here compared to getDefaultTimezone: It returns null in case nothing is defined and does not fall back to the system default.